deploy-board/deploy_board/webapp/templatetags/utils.py

```diff
@@ -605,7 +605,7 @@ def agentIcon(agentStats):
 
 @register.filter("hostButton")
 def hostButton(host):
-    if host['state'] == 'PENDING_TERMINATE' or host['state'] == 'TERMINATING':
+    if host['state'] in ('PENDING_TERMINATE', 'TERMINATING', 'PENDING_TERMINATE_NO_REPLACE'):
```

Comment:
May be worth defining ('PENDING_TERMINATE', 'TERMINATING', 'PENDING_TERMINATE_NO_REPLACE') in a common place, since it's used multiple times